src/BeaconDetector.mac.ts

```diff
@@ -0,0 +1,75 @@
+import noble from '@abandonware/noble';
+
+import AbstractBeaconDetector from './AbstractBeaconDetector';
+import logger from './logger';
+import { Config, Peripherial } from './model';
+
+// Mac specific beacon device ID.
+const BEACON_DEVICE_ID = '4c000215';
```

Comment:
this is a platform specific magic number, right? how does one find out the right one?

Reply:
By reverse engineering the binary data of the beacons we use